docs/configuration_options.rst

```diff
@@ -61,6 +61,20 @@ Connector
   * Default: 100
   * Importance: low
 
+``conn.timeout.ms``
+  How long to wait (in milliseconds) when establishing a connection to the Elasticsearch server.
+
+  * Type: int
+  * Default: 1000
+  * Importance: low
+
+``read.timeout.ms``
+  How long to wait (in milliseconds) for the Elasticsearch server to send responses.
+
+  * Type: int
```

Comment:
should be long

Reply:
The internal client only accepts `int`s for these configs. Didn't want to cast `long`s to `int`s internally.

Reply:
I see. Yeah, we don't want to cast for sure. Thanks for making other naming fixes. Your changes look good to me!